webpack.common.js

```diff
@@ -90,9 +90,18 @@ module.exports = env => {
             path.resolve(__dirname, 'node_modules/@patternfly/patternfly/assets/images'),
             path.resolve(__dirname, 'node_modules/@patternfly/react-styles/css/assets/images'),
             path.resolve(__dirname, 'node_modules/@patternfly/react-core/dist/styles/assets/images'),
-            path.resolve(__dirname, 'node_modules/@patternfly/react-core/node_modules/@patternfly/react-styles/css/assets/images'),
-            path.resolve(__dirname, 'node_modules/@patternfly/react-table/node_modules/@patternfly/react-styles/css/assets/images'),
-            path.resolve(__dirname, 'node_modules/@patternfly/react-inline-edit-extension/node_modules/@patternfly/react-styles/css/assets/images')
+            path.resolve(
+              __dirname,
+              'node_modules/@patternfly/react-core/node_modules/@patternfly/react-styles/css/assets/images'
+            ),
+            path.resolve(
+              __dirname,
+              'node_modules/@patternfly/react-table/node_modules/@patternfly/react-styles/css/assets/images'
+            ),
+            path.resolve(
+              __dirname,
+              'node_modules/@patternfly/react-inline-edit-extension/node_modules/@patternfly/react-styles/css/assets/images'
+            ),
           ],
           use: [
```

Comment:
I ran into an issue where using jpg and similar images as background images in css files wasn't working. Took me a while but I was able to hunt it down - looks like those new [asset modules](https://webpack.js.org/guides/asset-modules/) are applied by default, and since we're using the "old" asset loaders it was causing webpack to process these assets twice, and the resulting asset produced for images in stylesheets was corrupt. 

If we add `type: 'javascript/auto'` to this loader definition we get a module output where the path should be, which we can then just add `esModule: false,` to the options to get the intended path back.

Reply:
Hey @seanforyou23 . Sorry to be coming back to this so late, and I know you're elsewhere now so it's cool if you can't or don't want to respond.

I'm not able to reproduce the issue you decribe here. With the existing webpack config, I can put a .png file in `app/src/bgimages` and use it as a background in my CSS and it works fine with or without the options you mentioned. I do have to do a full `npm run build` for the new images to be picked up, though.

I think I'm going to say that if there's still an issue here it may fall outside the scope of this PR? If you feel like following up I'm happy to open another issue and fix it. 🍻